crawl-ref/source/dat/descript/monsters.txt

```diff
@@ -1954,6 +1974,12 @@ program bug
 A ravenous and incredibly buggy monster. Please report its existence to the
 DevTeam.
 %%%%
+putrid mouth
+
+A starved spirit whose mouth is a pit of rot and emits vile fumes. Its putrid
+maw cannot eat the food it finds, so it moans with rage and desparate hunger.
+The filth on its claws is withering.
```

Comment:
Is this last sentence mechanically significant? I wonder if we can cut it, since we describe monster AFs in the rest of xv anyway.